### PR TITLE
Add end of term date to SQL output

### PR DESCRIPTION
**Pull request by kevincianfarini**

### CODE REVIEW

File: src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt [hunk 1]
```diff
@@ -114,6 +118,7 @@ private fun Any?.sqlEscape(): String = when (this) {
   null -> "NULL"
   is String -> "'${this.replace("'", "''")}'"
   is Number -> toString()
+  is LocalDate -> "'$year-${monthNumber.toString().padStart(2, padChar = '0')}-$dayOfMonth'"
```
Comment (kevincianfarini) [inline src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt:121]: This is super naive, but kotlinx datetime is in pretty early beta and does not yet support date formatting.

Comment (EvanHahn) [inline src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt:121]: Should `$dayOfMonth` also be padded?

**Review by EvanHahn — APPROVED** [review]
LGTM other than one small comment.

File: src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt (hunk 1 above)
Comment (kevincianfarini) [reply src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt:121]: Yes. Let me add a test for it too.

Comment (kevincianfarini) [reply src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt]: Great call out.

**Pull request merged**